stable/fairwinds-insights/values.yaml

```diff
@@ -5,6 +5,9 @@ image:
 # -- Installation code provided by Fairwinds.
 installationCode:
 
+# -- Name of secret containing INSTALLATION_CODE
+installationCodeSecret:
```

Comment:
this needs to be added in docs in README

Reply:
done!